webrtc/videoroom/assets/src/room_ui.ts

```diff
@@ -76,6 +76,17 @@ export function addVideoElement(
   audio.muted = muted;
 }
 
+export function setParticipantsNamesList(
+  participantsNames: Array<string>
+): void {
+  const participantsNamesList = document.getElementById(
+    "participants-names-list"
+  ) as HTMLDivElement;
+
+  participantsNamesList.innerHTML =
```

Comment:
Use `innerText`. I don't know though if list of participants should be displayed in a row instead of column.